src/Extensions/Diagnostics.HealthChecks/Internal/RestClientWrapper.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Client;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	internal class RestClientWrapper
+	{
+		private IServiceFabricClient? m_client;
+
+		private readonly Uri? m_clusterEndpoint;
```

Comment:
Why this one nullable?

Reply:
beucase there is a ctor which accepts only client

Reply:
Generally it will be changed in next PR with more generic wrappers

Reply:
One thing, if something will be addressed in a separate PR, we usually add a TODO comment with the work item ID so it's tracked, for example: https://office.visualstudio.com/OC/_git/OmexMarketplace?path=%2Fsrc%2FOmexAddInsLicensing%2FAddInsLicensing.AcquireExperience%2FExperience%2FOutlookExperience.cs&version=GBmaster&line=106&lineEnd=107&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
Ok, done https://github.com/mvxxx/Omex/blob/rest-publisher/src/Extensions/Diagnostics.HealthChecks/Internal/RestClientWrapper.cs#L10